### ISSUE TITLE
Support arm64ec targets

### ISSUE THREAD

**Issue by TheJCAB** [microsoft/vscode-cmake-tools#2453]
### Brief Issue Summary

ARM64EC requires a different visual studio configuration. This documentation describes what to put in the preset file: https://devblogs.microsoft.com/cppblog/arm64ec-support-in-visual-studio/

```
 "architecture": { 
    "value": "arm64ec", 
    "strategy": "external"
 }
```

But when I try to build, the VS configuration used underneath (LIB environment variable, for instance) seems to point to x86.

### CMake Tools Diagnostics

I'd rather not post here. I can provide them on demand.

### Debug Log

No log. The feature seems to not be supported. I searched for "arm64ec" in the entire issue database and repo, and found no hits. This is a request to implement this (or to set me straight if I'm mistaken in any way).

**Comment by bobbrow** [comment]
Yes, you are correct when you say that it seems not to be supported.  It is not currently supported.  We can use this issue to track adding support for it.
